### DISCUSSION TITLE
Extended Industry Standard Architecture (EISA)

### DISCUSSION THREAD

**Discussion by ppgrainbow** [86Box/86Box#3870] [Ideas]
First of all, I would like to thank OBattler and his team for all of the hard work and progress on the 86Box and I really appreciate on continuing to develop new features that will be available in future builds of 86Box!

Here is something that I would like to see in 86Box v4 - the Extended Industry Standard Architecture or EISA. The standard was announced by a consortium of PC clone vendors- the Gang of Nine* as a alternative to IBM proprietary Micro Channel architecture (MCA) in its PS/2 series.

EISA had a short shelf life lasting nearly three years when it was succeeded by the VESA Local Bus and PCI.

Here are some notable machines that used EISA boards:

* AMI Enterprise-II
* Compaq Deskpro 486
* Compaq SystemPro
* Olivetti CP486
 
Here is some information regarding EISA on Wikipedia: https://en.wikipedia.org/wiki/Extended_Industry_Standard_Architecture

Even though 86Box has no support for EISA right now, I'm only making this feature request that I would like to see support for EISA in future releases - paving the way for support for some early 486 motherboards with EISA support.

In order to provide support for EISA chipsets, EISA graphics cards and EISA hard disk controllers, it would be wonderful to provide documentation and how it can be emulated.

Thoughts on EISA, anyone?

**DISCLAIMER:** The Gang of Nine, the informal name that was given to the consortium of personal computer manufacturing companies that together created the EISA bus lead by Compaq Computer Corporation. The members of the Gane of Nine were in alphabetical order:

* AST Research, Inc.
* Compaq Computer Corporation
* Hewlett-Packard Company
* NEC Corporation
* Olivetti
* Seiko Epson Corporation
* Tandy Corporation
* WYSE
* Zenith Data Systems

**Comment by ppgrainbow** [comment]
Here are some EISA-based hardware that I found:

* Adaptec AHA-1740: https://storage.microsemi.com/en-us/support/_eol/scsi_host_eisa/aha-1740/
* ELSA Winner 1000 (16-bit ISA and 32-bit EISA): https://stason.org/TULARC/pc/graphics-cards/E-H/ELSA-INC-XVGA-WINNER-1000-ISA-EISA-BUS.html

**Comment by Altheos** [comment]
I think your request is more or less request #377 duplicate.

**Comment by ppgrainbow** [comment]
> 
> 
> I think your request is more or less request #377 duplicate.

Holy crap, that post that you linked to is over two years ago.

What I was actually talking about is not just the addition of EISA support, but the feature request of hardware that uses a 32-bit EISA slot as I mentioned above.

**Comment by dhrdlicka** [comment]
> I think your request is more or less request #377 duplicate.

This feature request is more specific than the one you linked to, so it's not a duplicate.

**Comment by ppgrainbow** [comment]
> 
> 
> > I think your request is more or less request #377 duplicate.
> 
> This feature request is more specific than the one you linked to, so it's not a duplicate.

Thanks for the clarification.

Now since this is not a duplicate, can this feature be marked for 86Box v4.0 milestone?

**Comment by dhrdlicka** [comment]
Let's see how the version/release discussion ends up first.

**Comment by ppgrainbow** [comment]
> 
> 
> Let's see how the version/release discussion ends up first.

That sounds like a good idea. :)

By the way, are there any sound cards that use a 32-bit EISA bus? Just curious

**Comment by ppgrainbow** [comment]
Oh yeah, I have also found a good datasheet on the Intel 82350 chipset that supports EISA: https://www.datasheets360.com/pdf/-2372920747046840264

**Comment by davidrg** [comment]
EISA specification over here: https://www.os2museum.com/wp/eisa-specification-3-1/

**Comment by ppgrainbow** [comment]
Thank you for the help! :)

Let's hope that EISA support gets implemented soon! ^_^

**Comment by jriwanek** [comment]
For note, we also have detailed specs on the 82350 chipset, it's all down to time at this point,
